components/app-launcher/index.jsx

```diff
@@ -31,6 +31,7 @@ const defaultProps = {
 	assistiveText: {
 		trigger: 'Open App Launcher',
 	},
+	ariaHideApp: true,
```

Comment:
React modal has this prop to "opt out" of setAppElement. I think we should use that as well.